src/pyOpenMS/setup.py

```diff
@@ -256,7 +273,8 @@
 	install_requires=[
           'numpy<=1.26.4',
           'pandas',
-          'matplotlib>=3.5'
+          'matplotlib>=3.5',
+          'py-cpuinfo' ## Needed to check if our CPU supports AVX
```

Comment:
Hmm not sure if this will install before setup.py will run. You might need to switch to a pyproject.toml.

Reply:
ah. Couldn't it also be added in a requirements.txt file?

Reply:
Yes but remember, the user will not see the requirements anywhere.
When they run `pip install pyopenms` now, it will fail unless they can foresee that cpuinfo is required. This might break quite some automated installations.